yjit_codegen.c

```diff
@@ -2483,6 +2485,49 @@ gen_opt_getinlinecache(jitstate_t *jit, ctx_t *ctx)
     return YJIT_END_BLOCK;
 }
 
+static codegen_status_t
```

Comment:
Could you add a brief comment explaining what `getblockparamproxy` does? Still kind of unclear on that 😅

Reply:
@XrXr Maybe you should. Happy to proofread your post if you do :)